taichi/ui/backends/vulkan/renderable.h

```diff
@@ -42,6 +42,7 @@ struct RenderableConfig {
       taichi::lang::TopologyType::Triangles};
   taichi::lang::PolygonMode polygon_mode{taichi::lang::PolygonMode::Fill};
   VertexAttributes vbo_attrs{VboHelpers::all()};
+  bool point_instancing{false};
```

Comment:
How about just calling this `vertex_input_rate_instance`? So that the meaning of this flag is more obvious (and generic).